life/modernized/09-chapter-9.adoc

```diff
@@ -31,7 +31,10 @@ to Friends and others in Holland and Germany.
 Of this journey he has preserved an account,
 which is so full of lively interest and instruction,
 that the narrative cannot be better conveyed than in his own language.
-It was first published in the year 1694, as follows:
+It was first published in the year 1694, as follows.
```

Comment:
I removed the colon after "as follows" because the asterisks seem to break it up too much.